docs/modular-resources/advanced/create-subtype.md

```diff
@@ -4,76 +4,83 @@ linkTitle: "New API Subtype"
 weight: 30
 type: "docs"
 tags: ["rdk", "extending viam", "modular resources", "API"]
-description: "Define a new resource subtype using an existing resource API."
+description: "Define a new resource subtype and API for a resource that does not fit into existing component or service subtypes."
```

Comment:
Ooh that description got incorrected--good it's getting fixed. But, I think "subtype and API" is redundant; how about:
```suggestion
description: "Define a new API for a resource that does not fit into existing component or service subtypes."
```